### ISSUE TITLE
Update Jackson to 2.13

### ISSUE THREAD

**Issue by jeffmaury** [redhat-developer/intellij-kubernetes#370]
related to #354 
depends on #272 

See https://security.snyk.io/vuln/SNYK-JAVA-COMFASTERXMLJACKSONCORE-2421244

**Comment by adietish** [comment]
@jeffmaury I dont depend on jackson, it is a transient dependency. 
```
+--- io.fabric8:kubernetes-model:5.7.2
|    +--- io.fabric8:kubernetes-model-core:5.7.2 (*)
|    +--- io.fabric8:kubernetes-model-rbac:5.7.2 (*)
|    +--- io.fabric8:kubernetes-model-admissionregistration:5.7.2 (*)
|    +--- io.fabric8:kubernetes-model-apps:5.7.2 (*)
|    +--- io.fabric8:kubernetes-model-autoscaling:5.7.2 (*)
|    +--- io.fabric8:kubernetes-model-apiextensions:5.7.2 (*)
|    +--- io.fabric8:kubernetes-model-batch:5.7.2 (*)
|    +--- io.fabric8:kubernetes-model-certificates:5.7.2 (*)
|    +--- io.fabric8:kubernetes-model-coordination:5.7.2 (*)
|    +--- io.fabric8:kubernetes-model-discovery:5.7.2 (*)
|    +--- io.fabric8:kubernetes-model-events:5.7.2 (*)
|    +--- io.fabric8:kubernetes-model-extensions:5.7.2 (*)
|    +--- io.fabric8:kubernetes-model-networking:5.7.2 (*)
|    +--- io.fabric8:kubernetes-model-metrics:5.7.2 (*)
|    +--- io.fabric8:kubernetes-model-policy:5.7.2 (*)
|    +--- io.fabric8:kubernetes-model-scheduling:5.7.2 (*)
|    +--- io.fabric8:kubernetes-model-storageclass:5.7.2 (*)
|    +--- io.fabric8:openshift-model:5.7.2 (*)
|    \--- com.fasterxml.jackson.core:jackson-annotations:2.11.2
```
To bump jackson we'd have to upgrade kubernetes-client.

Kubernetes-client updated jackson 2.11 -> 2.13 in https://github.com/fabric8io/kubernetes-client/commit/558092b9f45fe8b0aea2d0b1dcc95e434c531eaf which was then released as kubernetes-client 5.9.0 (https://github.com/fabric8io/kubernetes-client/commit/5f5c8d3f76375ffc0bccfc3c4f417beaac031024).
In other words, to bump jackson to 2.13 we'd have to bump kubernetes-client to >=5.9.0

**Comment by jeffmaury** [comment]
> @jeffmaury, @lstocchi: should we upgrade the kubernetes-client in the next sprint or hold back?

No need to hold as the update is a minor one and has already been done in other plugins

**Comment by adietish** [comment]
@jeffmaury kubernetes-client 6.x will break a lot of APIs, look at their migration guide: https://github.com/fabric8io/kubernetes-client/blob/master/doc/MIGRATION-v6.md

**Comment by adietish** [comment]
I'll takle #370 before migrating to the latest 5.x
